src/store/juju/selectors.ts

```diff
@@ -184,6 +184,16 @@ export const getControllerData = createSelector(
   (sliceState) => sliceState.controllers
 );
 
+export const getControllersCount = createSelector([slice], (sliceState) => {
+  const controllerData = sliceState.controllers;
+  return controllerData
+    ? Object.entries(controllerData).reduce(
+        (count, [, controllers]) => count + controllers.length,
```

Comment:
You could use `Object.values` here so that you don't have to destructure the array.

```suggestion
    ? Object.values(controllerData).reduce(
        (count, controllers) => count + controllers.length,
```

Reply:
Good catch! Changed it in my latest commit.